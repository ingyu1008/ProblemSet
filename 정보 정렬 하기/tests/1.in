[[517862038, 'dhpwivzkoqpbqxofnckptlyxrflgpppzvkjvkimkurbpuiinvk', 3316579], [161444759, 'uwmshrvemyrumijhlpapmqnqdsaqninhcicfckwxxnnzlpvqed', 874031135], [552367699, 'lpebqiqevszuourjhpiegfkyruuchqildmgfvxysuuuwzqvdri', 517369338], [202730872, 'foiaqgjskfpzjueealpdiqorkucjgwtsujherphboxerfrztzs', 266498688], [850477138, 'uxbeilufkfikiyywlanhrpgpvneobhloxaxfsbkfcvuarcnnqr', 299363503], [637263897, 'gajotgusjlbhkvxrfqgxdhmwszqmbostcfypmfxpakorbqowds', 888643922], [189533944, 'zodqklikbwjvsmbikejabjiptocrxndiekksaggdtfznbvkmfv', 283187429], [801996300, 'jvlczpalvvkjqrvcbpunjdnftlklktjtbcpuqlxtkqrxslrxva', 633563307], [905973071, 'nbqdnjzndhkudbblwufzhuwcbkycnxipgcuzvipmffxctwvdff', 431564871], [214434670, 'wendjehfiygoopbetadeahkonyujeaznwmbwoggtgommvysxow', 275713819]]