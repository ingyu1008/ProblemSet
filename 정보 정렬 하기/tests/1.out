[
    [
        161444759,
        189533944,
        202730872,
        214434670,
        517862038,
        552367699,
        637263897,
        801996300,
        850477138,
        905973071
    ],
    [
        517862038,
        202730872,
        637263897,
        801996300,
        552367699,
        905973071,
        161444759,
        850477138,
        214434670,
        189533944
    ],
    [
        637263897,
        161444759,
        801996300,
        552367699,
        905973071,
        850477138,
        189533944,
        214434670,
        202730872,
        517862038
    ]
]